scripts/bin/update-checksum

```diff
@@ -57,29 +57,42 @@ for package in "${@}"; do
 
 	# Calculate new checksum in subshell.
 	echo
-	new_checksum=$(
+	new_checksum=($(
 		source "${buildsh_path}" 2>/dev/null
+		for url in ${TERMUX_PKG_SRCURL[@]}; do
+			dl_tmpdir=$(mktemp -d "${TMPDIR-/tmp}/termux.src.dl.XXXXXXXX")
 
-		dl_tmpdir=$(mktemp -d "${TMPDIR-/tmp}/termux.src.dl.XXXXXXXX")
+			if [ -n "${dl_tmpdir}" ] && [ -d "${dl_tmpdir}" ]; then
+				if ! curl --fail --location --retry 3 --output "${dl_tmpdir}/source-bundle" \
+					"$url"; then
+					rm -rf "${dl_tmpdir}"
+				fi
 
-		if [ -n "${dl_tmpdir}" ] && [ -d "${dl_tmpdir}" ]; then
-			if ! curl --fail --location --retry 3 --output "${dl_tmpdir}/source-bundle" \
-				"${TERMUX_PKG_SRCURL[0]}"; then
-				rm -rf "${dl_tmpdir}"
-			fi
+				if [ -f "${dl_tmpdir}/source-bundle" ]; then
+					sha256sum "${dl_tmpdir}/source-bundle" | awk '{ print $1 }'
+				fi
 
-			if [ -f "${dl_tmpdir}/source-bundle" ]; then
-				sha256sum "${dl_tmpdir}/source-bundle" | awk '{ print $1 }'
+				rm -rf "${dl_tmpdir}"
 			fi
-
-			rm -rf "${dl_tmpdir}"
-		fi
-	)
+		done
+	))
 
 	if [ -n "${new_checksum}" ]; then
```

Comment:
That's an array now.
Also prefer using arithmetic evaluation for a null length check.
```suggestion
	if (( ${#new_checksum[@]} )); then
```